schemas/stsci.edu/asdf/core/externalarray-1.0.0.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/core/externalarray-1.0.0"
+tag: "tag:stsci.edu:asdf/core/externalarray-1.0.0"
+title: Point to an array-like object in an external file.
+
+type: object
+properties:
+  source:
+    type: string
+  target:
+    anyOf:
+      - type: integer
+      - type: string
+  datatype:
+    type: string
+  shape:
+    type: array
+    items:
+      anyOf:
+      - type: integer
+        minimum: 0
+      - enum: ['*']
```

Comment:
It's not clear to me what purpose this serves.

Reply:
🤷‍ I have no idea either. I will remove it and see if it breaks.